.github/workflows/superset-e2e.yml

```diff
@@ -74,8 +79,10 @@ jobs:
           cypress-install
     - name: Run Cypress
       uses: apache-superset/cached-dependencies@b90713b
+      with:
+        parallel: true
+        record: true
```

Comment:
`parallel` and `record` settings here are unnecessary, as they're already enabled in the CI bash script.